### ISSUE TITLE
Certain character sequence as node label breaks rendering

### ISSUE THREAD

**Issue by rogerbock** [xflr6/graphviz#86]
My guess is lang.py's quote() function is failing to add quotes around something it should be quoting.  In the code provided below, the string in question is 5.0 written using Eastern Arabic numerals.

Code to reproduce:

```python
from graphviz import Digraph
graph = Digraph()
graph.node('test', '٥.٠')
graph.render()
```

Resulting error:
```text
Error: Digraph.gv: syntax error in line 2 near '.'
Traceback (most recent call last):
  File "test.py", line 4, in <module>
    graph.render()
  File "miniconda3/lib/python3.7/site-packages/graphviz/files.py", line 209, in render
    quiet=quiet)
  File "miniconda3/lib/python3.7/site-packages/graphviz/backend.py", line 207, in render
    run(cmd, capture_output=True, cwd=cwd, check=True, quiet=quiet)
  File "miniconda3/lib/python3.7/site-packages/graphviz/backend.py", line 173, in run
    output=out, stderr=err)
graphviz.backend.CalledProcessError: Command '['dot', '-Tpdf', '-O', 'Digraph.gv']' returned non-zero exit status 1. [stderr: b"Error: Digraph.gv: syntax error in line 2 near '.'\n"]
```

Output dot file that contains error:
```text
digraph {
        test [label=٥.٠]
}
```

**Comment by xflr6** [comment]
Thanks for the detailed report. Can you check if 4858bf7216dc1cc6369604ecf50a7e6cfeb0977c resolves this for you?

**Comment by rogerbock** [comment]
It works.  Thank you for the quick response!

**Comment by xflr6** [comment]
Tanks. Fixed in [0.13.1 release](https://pypi.org/project/graphviz/0.13.1/#files) (6f5eee77ebd5a66bce8db96bcfeb02a6e616fab4).
